db/schema.rb

```diff
@@ -153,7 +153,7 @@
     t.string "status"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.index ["nomis_staff_id"], name: "index_pom_details_on_nomis_staff_id"
+    t.index ["nomis_staff_id"], name: "index_pom_details_on_nomis_staff_id", unique: true
```

Comment:
Does the table already have this unique constraint or do we need to add it in a migration?

Reply:
Need to add it in a migration!